components/blocks/subscribe.tsx

```diff
@@ -0,0 +1,34 @@
+import { Template } from "tinacms";
+import { Container } from "../util/container";
+import { SubNewsLetters } from "./subNewsLetters";
+
+export const Subscribe = () => {
+  return (
+    <div
+      // eslint-disable-next-line tailwindcss/no-arbitrary-value
+      className={
+        "flex h-80 w-full items-center bg-[url('/images/thumbs/subscribeBackground.png')] bg-cover bg-center bg-no-repeat text-center font-light after:absolute"
```

Comment:
```suggestion
      className={
        "flex h-80 w-full items-center bg-subscribe-background bg-cover bg-center bg-no-repeat text-center font-light after:absolute"
```

if you add the image to the Tailwind config file, you can avoid disabling this warning 

We have done this for a bunch of other images:
https://github.com/SSWConsulting/SSW.Website/blob/3bc101637f3d268775f5c26ee502ddd015ac7575/tailwind.config.js#L374-L383

Reply:
**Done** - Thanks for the advice😃